src/parser/config/nodeTypesConfig.json

```
@@ -3029,7 +3029,7 @@
     },
     {
         "type_name": "binary_expression_xor",
-        "category": "logical_binary_expression",
+        "category": "",
```

should this be binary_expression?
```suggestion
        "category": "binary_expression",
```